File: 프로그래머스/unrated/181838. 날짜 비교하기/날짜 비교하기.py
def solution(date1, date2):
    if date1[0] < date2[0]:
        return 1
    elif date1[0] > date2[0]:
        return 0
    if date1[1] < date2[1]:
        return 1
    elif date1[1] > date2[1]:
        return 0
    if date1[2] < date2[2]:
        return 1
    elif date1[2] > date2[2]:
        return 0
    return 0

File: 프로그래머스/0/181838. 날짜 비교하기/날짜 비교하기.java
class Solution {
    public int solution(int[] date1, int[] date2) {
        int date1Num = date1[0] * 365 + date1[1] * 30 + date1[2];
        int date2Num = date2[0] * 365 + date2[1] * 30 + date2[2];
        
        return date1Num >= date2Num ? 0 : 1;
    }
}